signingscript/src/signingscript/sign.py

```diff
@@ -286,13 +286,16 @@ def get_autograph_config(
     """
     file_base, file_extension = os.path.splitext(orig_path)
 
-    if not file_extension == ".xpi":
+    if file_extension not in (".xpi", ".zip"):
         raise SigningScriptError("Expected a .xpi")
 
-    id = _langpack_id(orig_path)
+    ext_id = _extension_id(orig_path, fmt)
     log.info("Identified {} as extension id: {}".format(orig_path, id))
+    kwargs = {'extension_id': ext_id}
+    if 'langpack' not in fmt:
+        kwargs['keyid'] = fmt
```

Comment:
Let's make this explicit in the passwords file.

Reply:
Ok. I think the latest commit does that.